### PR TITLE
George login

### PR DESCRIPTION
**Pull request by georgezsiga**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,6 +2,11 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.greenfox.tribesoflagopusandroid">
 
+    <!-- To auto-complete the email text field in the login form with the user's emails -->
+    <uses-permission android:name="android.permission.GET_ACCOUNTS" />
+    <uses-permission android:name="android.permission.READ_PROFILE" />
+    <uses-permission android:name="android.permission.READ_CONTACTS" />
```
Comment (adamgyulavari) [inline app/src/main/AndroidManifest.xml]: @georgezsiga why do you need these permissions? remove them

File: app/src/main/res/layout/activity_login.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    <EditText
+        android:id="@+id/editText2"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="16dp"
+        android:layout_marginRight="16dp"
+        android:layout_marginTop="16dp"
+        android:ems="10"
+        android:hint="@string/User_password"
```
Comment (adamgyulavari) [inline app/src/main/res/layout/activity_login.xml]: @georgezsiga we use lowercase_dash_separated names for strings

File: app/src/test/java/com/greenfox/tribesoflagopusandroid/LoginActivityTest.java [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        assertEquals(password.getText().toString(), login.preferences.getString("Password", ""));
+    }
+
+    @Test
+    public void startMainActivity() {
+        login.findViewById(R.id.button2).performClick();
+        Intent expectedIntent = new Intent(login, MainActivity.class);
+        assertEquals(expectedIntent.getClass(), shadowOf(login).getNextStartedActivity().getClass());
+    }
+}
```
Comment (adamgyulavari) [inline app/src/test/java/com/greenfox/tribesoflagopusandroid/LoginActivityTest.java]: @georgezsiga have always a new line at the end of all files

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (georgezsiga) [reply app/src/main/AndroidManifest.xml]: done

File: app/src/main/res/layout/activity_login.xml (hunk 2 above)
Comment (georgezsiga) [reply app/src/main/res/layout/activity_login.xml]: done

File: app/src/test/java/com/greenfox/tribesoflagopusandroid/LoginActivityTest.java (hunk 3 above)
Comment (georgezsiga) [reply app/src/test/java/com/greenfox/tribesoflagopusandroid/LoginActivityTest.java]: done

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**